peachjam/templates/peachjam/_court_list.html

```diff
@@ -1,13 +1,13 @@
 {% load peachjam i18n %}
-{% if grouped_courts %}
+{% if court_classes %}
   <div class="courts__column-wrapper mb-3">
-    {% for court_class, courts in grouped_courts.items %}
+    {% for court_class in court_classes %}
       <div class="courts__item mb-3">
         <h4 class="heading-underlined">{{ court_class }}</h4>
-        {% if courts|length > 0 %}
-          {% for item in courts %}
+        {% if court_class.courts.all %}
+          {% for court in court_class.courts.all %}
             <div>
-              <a href="{% url 'court' item.code %}">{{ item.title }}</a>
+              <a href="{% url 'court' court.code %}">{{ court }}</a>
```

Comment:
Dangerous to rely on the stringified form of the court, which may not in future be the title. Better to be explicit.

```suggestion
              <a href="{% url 'court' court.code %}">{{ court.title }}</a>
```